redis/src/main/scala/zio/redis/TestExecutor.scala

```diff
@@ -257,7 +260,48 @@ private[redis] final class TestExecutor private (
         val key   = input.head.asString
         val value = input(1).asString
         strings.put(key, value).as(Replies.Ok)
+      case api.HyperLogLog.PfAdd.name =>
+        val key    = input.head.asString
+        val values = input.tail.map(_.asString)
+        STM.ifM(forAll(Chunk.single(key))(isSet))(
+          for {
+            oldValues <- hyperLogLogs.getOrElse(key, Seq.empty[String])
+            ret        = if (oldValues == values) 0L else 1L
+            _         <- hyperLogLogs.put(key, values)
+          } yield {
+            RespValue.Integer(ret)
+          },
+          STM.succeedNow(Replies.WrongType)
+        )
 
+      case api.HyperLogLog.PfCount.name =>
+        val keys = input.map(_.asString)
+        STM.ifM(forAll(keys)(isSet))(
+          STM
+            .foldLeft(keys)(Seq.empty[String]) { (bHyperLogLogs, nextKey) =>
+              hyperLogLogs.getOrElse(nextKey, Seq.empty[String]).map { currentSet =>
+                bHyperLogLogs ++ currentSet
+              }
+            }
+            .map(vs => RespValue.Integer(vs.size.toLong)),
+          STM.succeedNow(Replies.WrongType)
+        )
+      case api.HyperLogLog.PfMerge.name =>
+        val key    = input.head.asString
+        val values = input.tail.map(_.asString)
+        STM.ifM(forAll(values ++ Chunk.single(key))(isSet))(
+          for {
+            sourceValues <- STM.foldLeft(values)(Seq.empty[String]) { (bHyperLogLogs, nextKey) =>
```

Comment:
I'd use chunk instead of a Seq.

Reply:
Chunk.empty  generice type is Nothing,can not use ++ with Chunk String